test/functionalTest/cases/2584_service_path_header_in_notifs/service_path_header_in_notifs.test

```diff
@@ -0,0 +1,196 @@
+# Copyright 2017 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Do we really need a new .test for this when the large ammount of notifications in default SP we have in another tests will ensure the functionality is correct? I does't suggest to remove it (as it is already written) just a remark to think twice next time in similar situations ;)

NTC